docs/core/tutorials/using-on-windows.md

```diff
@@ -121,17 +121,19 @@ A solution using only .NET Core projects
 
 2. In the **App** project, open the context menu for the **References** node and choose **Add**,  **Reference**. 
 
-3. In the **Reference Manager** dialog, check **Library** under the **Projects**, **Solution** node, and then click **OK** 
+3. In the **Reference Manager** dialog, check **Library** under the **Projects**, **Solution** node, and then click **OK**
 
-4. Open the context menu for the **References** node and choose **Restore Packages**.
+4. In the **App** project, open the `project.json` file, and replace `"Library": "1.0.0-*"` with `"Library": {"target": "project", "version": "1.0.0-*"}`.
```

Comment:
We're trying to discourage the "1.0.0-*" reference syntax. Can you replace 

`"Library": {"target": "project, "version": "1.0.0-*"}`

with:

`"Library": {"target": "project"}`

Thank you.